### DISCUSSION TITLE
[Art competition] Fully Homomorphic Encryption

### DISCUSSION THREAD

**Discussion by jj2379** [prosyslab-classroom/cs348-information-security#399] [Art competition]
![DALL·E 2023-05-22 15 13 51](https://github.com/prosyslab-classroom/cs348-information-security/assets/38258278/dfd187bd-7fdf-41f6-8c71-ccba8ba29420)

## Short Description
For me, Fully Homomorphic Encryption means "surrealist painting of a cloud with closed eyes and a lot of hands covering the screen"

## Long Description (up to 3 sentences)
FHE enables calculation even in an encrypted state, and through this, you can editing file uploaded to the cloud, preventing an other person from what was the change in the content. I thought of this as a hand(FHE) covering the eyes of the cloud(web cloud) with a hand touching the cloud(user).

**Comment by KihongHeo** [comment]
Good. In my experience, this AI is not good at drawing hands..
